docs/en/ingest-management/integrations/managed-integrations-content.asciidoc

```diff
@@ -0,0 +1,20 @@
+[[managed-integrations-content]]
+= Managed integrations content
+
+++++
+<titleabbrev>Managed integrations content</titleabbrev>
+++++
+
+Most integration content installed by {fleet} isn’t editable. This content is tagged with a **Managed** badge in the {kib} UI. Managed content itself cannot be edited or deleted, however managed visualizations, dashboards, and saved searches can be cloned.   
+
+[role="screenshot"]
+image::images/system-managed.png[An image of the new managed badge.]
+
+With dashboards the cloned content is unlinked with the original reference configurations, so becomes independent from the original managed content. For example, with a cloned dashboard, the cloned panels become entirely self-contained copy without any dependencies on the original configuration. Clones can be customized and modified without accidentally affecting the original.
+
+[NOTE]
+====
+The cloned managed content retains the managed badge, but is independent from the original. 
+====
+
+With managed content relating to specific visualization editor such as Lens, TSVB, and Maps, the clones retain the original reference configurations. To clone the visualization, view it in the editor then becoming to make edits. Once finished editing you are prompted to save the edits as a new visualization. The same applies to editing any saved searches in a managed visualization.  
```

Comment:
Typo?

> view it in the editor then becoming to make edits

Reply:
Oh yes it was supposed to say begin not becoming. Corrected is latest commit.